spec/api/payment/void_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe Afterpay::API::Payment::Void do
+  describe '#call' do
+    let(:token) { 'some_token' }
+    let(:order_id) { '12345' }
+    let(:payment) { build(:payment, token: token) }
+
+    context 'with proper params' do
+      let(:raw_response) { JSON.generate(paymentState: 'VOIDED') }
+
+      before(:each) do
+        stub_request(:post, %r{api.us.afterpay.com/v2/payments/#{order_id}/void})
+          .to_return(
+            status: 201,
+            body: raw_response,
+            headers: { 'Content-Type' => 'application/json' }
+          )
+      end
+
+      it 'returns hashie mash' do
+        expect(described_class.call(payment: payment, order_id: order_id, amount: "12.11").body).to be_instance_of Hashie::Mash
+      end
+
+      it 'returns afterpay order id' do
+        expect(described_class.call(payment: payment, order_id: order_id, amount: "12.11").body).to include(:paymentState)
+      end
+    end
+
+    context 'client not void payment' do
+      let(:error_message) { 'Afterpay payment ID not found.' }
+      let(:raw_response) do
+        {
+          errorCode: 'invalid',
+          errorId: '5393ff621317c438',
+          message: error_message,
+          httpStatusCode: 404
+        }.to_json
+      end
+
+      before(:each) do
+        stub_request(:post, %r{api.us.afterpay.com/v2/payments/12/void})
```

Comment:
Can you create an `order_id` variable also in this context? Like you do in the `with proper params` context

Reply:
Done created an order_id variable with a different number